pkg/cmd/kind/version/version.go

```diff
@@ -58,7 +58,7 @@ const versionCore = "0.23.0"
 
 // versionPreRelease is the base pre-release portion of the kind CLI version per
 // Semantic Versioning 2.0.0
-const versionPreRelease = "alpha"
+var versionPreRelease = "alpha"
```

Comment:
.Not a fan of mutating global vars to implement unit tests

Reply:
Alternatively, we can skip this test if versionPreRelease == "". 
Open to other suggestions

Reply:
re-reading this, I see we've already done that for other fields.

What we should really do is make Version() a shim to version() passing the globals and test `version` for invariants, but we can leave that to a follow-up.

Reply:
(since we already had this global mutating pattern prior to this PR)

Reply:
SG, thanks for the approval. I can look into your suggestion of making Version() a shim in a follow up.